samtranslator/model/api/api_generator.py

```diff
@@ -27,7 +27,8 @@ class ApiGenerator(object):
     def __init__(self, logical_id, cache_cluster_enabled, cache_cluster_size, variables, depends_on,
                  definition_body, definition_uri, name, stage_name, endpoint_configuration=None,
                  method_settings=None, binary_media=None, cors=None, auth=None, access_log_setting=None,
-                 canary_setting=None, tracing_enabled=None):
+                 canary_setting=None, tracing_enabled=None, resource_attributes=None,
```

Comment:
https://github.com/awslabs/serverless-application-model/blob/9206b05f049ba7465d11737c42be9ccb23c94239/samtranslator/model/api/api_generator.py#L30-L31

Can you add definitions for the newly added params in function docstring?

Reply:
Good catch. Will add